### ISSUE TITLE
Missing questionnaires' button's text and they don't switch to next tab on click

### ISSUE THREAD

**Issue by diegolagoglez** [globaleaks/globaleaks-whistleblowing-software#4298]
### What version of GlobaLeaks are you using?

5.0.20, 5.0.21, 5.0.22

### What browser(s) are you seeing the problem on?

Chrome, Firefox

### What operating system(s) are you seeing the problem on?

Windows, Linux

### Describe the issue

After upgrade from version 5.0.19 to version 5.0.20, 21 and 22, using `apt` in Ubuntu 22.04, questionnaires' button's text is missing (text should be _Siguiente_) and the button doesn't switch to the next tab on click, as it can be seen in this screenshot:

![globaleaks-button](https://github.com/user-attachments/assets/e5b5dfe4-c42e-4ab6-b050-8ae02c43d1db)

Javascript console raises these errors, all of them in `core.mjs` file:

```js
ERROR TypeError: t[Symbol.iterator] is not a function
    Angular 2
    template form.component.html:1
    Angular 69
    RxJS 6
    Angular 7
    Je ng-idle-core.mjs:506
    Angular 13
    setIdleIntervalOutsideOfZone ng-idle-core.mjs:524
    Angular 3
    setIdleIntervalOutsideOfZone ng-idle-core.mjs:523
    Je ng-idle-core.mjs:510
    Angular 5
    Je ng-idle-core.mjs:506
    Angular 13
    setIdleIntervalOutsideOfZone ng-idle-core.mjs:524
    Angular 3
    setIdleIntervalOutsideOfZone ng-idle-core.mjs:523
ERROR TypeError: t[Symbol.iterator] is not a function
    Angular 2
    template form.component.html:1
    Angular 69
    RxJS 6
    Angular 20
    RxJS 31
```

These errors are always appearing, even when no interaction with the page.

Also, I think this error is related to [4190](https://github.com/globaleaks/globaleaks-whistleblowing-software/issues/4190).

**Comment by evilaliv3** [comment]
@msmannan00 : would you please check on this ?

**Comment by evilaliv3** [comment]
The problem can be tested with this questionnaire in Valencian language:
[AVAF con filtro previo.json](https://github.com/user-attachments/files/17659513/AVAF.con.filtro.previo.json)


[Screencast from 2024-11-07 10-48-44.webm](https://github.com/user-attachments/assets/58112534-f91c-4375-8b58-bdf651461e13)

**Comment by evilaliv3** [comment]
I think i may have found the bug with the questionnaire i loaded above. In that questionnaire the last step does not contain any question.

@diegolagoglez : do you have a similar situation in your questionnaire?

**Comment by JaviAlama** [comment]
You're right, the problem using Valencian still remains.

![image](https://github.com/user-attachments/assets/970d2be4-8e2b-4813-a7ef-1079f1e94b6e)

**Comment by diegolagoglez** [comment]
> I think i may have found the bug with the questionnaire i loaded above. In that questionnaire the last step does not contain any question.
> 
> @diegolagoglez : do you have a similar situation in your questionnaire?

Exactly. Our last tab in our questionnaire doesn't contain any question, it's only informative.

**Comment by evilaliv3** [comment]
I see @diegolagoglez 

The issue should be fixed in release 5.0.25 even if the step does not contain any question.

We suggest anyhow to use questions of type Terms and Conditions, where to ask acceptance of the terms.
